samples/bluetooth/broadcast_audio_source/overlay-bt_ll_sw_split.conf

```diff
@@ -14,11 +14,6 @@ CONFIG_BT_CTLR_ADV_ISO_PDU_LEN_MAX=155
 CONFIG_BT_CTLR_ADV_ISO_STREAM_MAX=2
 CONFIG_BT_CTLR_ISOAL_SOURCES=2
 
-# FIXME: Host needs CONFIG_BT_ISO_TX_MTU + 4 bytes for sequence number, and  optionally
-# additional + 4 bytes for timestamp when not using BT_ISO_TIMESTAMP_NONE in bt_iso_chan_send(),
-# otherwise Host tries to fragment ISO data.
-# When Host is fixed, CONFIG_BT_CTLR_ISO_TX_BUFFER_SIZE can inherit the
-# CONFIG_BT_ISO_TX_MTU value.
-#
-# Supports the highest SDU size required by any BAP LC3 presets (155)
+# Support the highest SDU size required by any BAP LC3 presets (155) + 8 bytes of HCI ISO Data
+# packet overhead (timestamp and HCI ISO Data packet header)
```

Comment:
The core spec states for Read Buffer Size V2:

> Note: The LE_ACL_Data_Packet_Length and ISO_Data_Packet_Length return parameters do not include the length of the HCI ACL Data packet header or the HCI ISO Data packet header respectively.

So this should only need to be 155 + 4 = 159, as the HCI ISO Data packet header (4 octets) are _not_ included in the buffer size. 

As I understand it, all the places where you have `+ 8`, it should be `+ 4`, except the places where you use this to allocate memory for HCI ISO Data packets

Reply:
As mentioned in the comment in the file, 8 bytes is for the 4 bytes HCI ISO Data packet header plus 4 bytes for the optional timestamp (commented as the required overhead) if used by the upper layer. If only additional 4 bytes is allocated, and upper layer uses API with timestamp, then SDU would be fragmented over HCI.

Reply:
yes @cvinayak and I got confused previously. @ivaniushkov fixed that miscalculation in the host a few months ago.

Reply:
I'm still confused :D 

Is `CONFIG_BT_CTLR_ISO_TX_BUFFER_SIZE` the size of the buffers returned from the HCI command? If so, then it should _not_ include the 4 bytes header.

Reply:
i stand corrected... the 8 bytes is for the following:
![image](https://github.com/zephyrproject-rtos/zephyr/assets/6350656/4d3597d9-0ace-480c-969f-944d72ed67fc)

Are we in sync? I will fix comments in this PR.

Reply:
As I understand it, the TX buffer size from the controller is everything after the first 4 octets of Header (Conn handle + Data_Total_Length, etc.).

So for example to send an SDU of size 100 the buffer needs to be at least 100 + sizeof(psn) + sizeof(iso_sdu_length + status flag) = 104, or 108 if it include the timestamp. 

Today `CONFIG_BT_ISO_TX_MTU` determines the size of the SDU (i.e. the maximum value of ISO_SDU_length), but not the timestamp or the PSN, etc.  (that is added by the `BT_ISO_SDU_BUF_SIZE` macro). 

So if we want to derive `CONFIG_BT_CTLR_ISO_TX_BUFFER_SIZE` from `CONFIG_BT_ISO_TX_MTU` then we either need to:
1) `CONFIG_BT_CTLR_ISO_TX_BUFFER_SIZE = CONFIG_BT_ISO_TX_MTU + BT_HCI_ISO_TS_DATA_HDR_SIZE (8)`
2) Modify `BT_ISO_SDU_BUF_SIZE` so that it does not contain the timestamp and then do `CONFIG_BT_CTLR_ISO_TX_BUFFER_SIZE = CONFIG_BT_ISO_TX_MTU + BT_HCI_ISO_DATA_HDR_SIZE (4)`

So actually you were correct in using 8, but for the wrong reason.

Reply:
> So actually you were correct in using 8 (with timestamp), but for the wrong reason.

Agreed. @Emil-Gydesen-Bose  is the follow reasonably correct?
```
# Support the highest SDU size required by any BAP LC3 presets (155) + 8 bytes of HCI ISO Data
# packet overhead (the Packet_Sequence_Number, ISO_SDU_Length, Packet_Status_Flag fields; and
# the optional Time_Stamp field, if supplied)
```